src/module/cp/peeper.py

```
@@ -208,6 +224,22 @@ def peeper_daily_update_job():
     Constants.log.info("[peeper] 每日榜单更新任务完成")
 
 
+def _collect_push_targets() -> list[str]:
+    """从配置中收集所有 obot_push_to 目标 uuid，经过 _generate_peeper_conf 校验"""
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)